frontend/public/components/volumes-table.tsx

```diff
@@ -52,83 +39,107 @@ const ContainerLink: React.FC<ContainerLinkProps> = ({name, pod}) => <span class
 </span>;
 ContainerLink.displayName = 'ContainerLink';
 
-const VolumeRow: React.FC<VolumeRowProps> = ({value, pod}) => {
-  const kind = _.get(getVolumeType(value.volume), 'id', '');
-  const loc = getVolumeLocation(value.volume);
-  const name = value.name;
-  const permission = value.readOnly ? 'Read-only' : 'Read/Write';
+const volumeRoColumnClasses = [
+  classNames('col-lg-2', 'col-md-3', 'col-sm-4', 'col-xs-5'),
+  classNames('col-lg-2', 'col-md-3', 'col-sm-4', 'col-xs-7'),
+  classNames('col-lg-2', 'col-md-2', 'col-sm-4', 'hidden-xs'),
+  classNames('col-lg-2', 'col-md-2', 'hidden-sm', 'hidden-xs'),
+  classNames('col-lg-2', 'col-md-2', 'hidden-sm', 'hidden-xs'),
+  classNames('col-lg-2', 'hidden-md', 'hidden-sm', 'hidden-xs'),
+  Kebab.columnClass,
+];
 
-  return <div>
-    {value.mounts.map((m: Mount, i: number) => <React.Fragment key={i}>
-      <div className="row">
-        <div className="col-lg-2 col-md-3 col-sm-4 col-xs-5">{name}</div>
-        <div className="col-lg-2 col-md-3 col-sm-5 col-xs-7 co-break-all co-select-to-copy">{_.get(m, 'mountPath', '-')} </div>
-        <div className={classNames('col-lg-2 col-md-2 col-sm-3 hidden-xs co-break-all', { 'co-select-to-copy': _.get(m, 'subPath') })}>
-          {_.get(m, 'subPath', '-')}
-        </div>
-        <div className="col-lg-2 col-md-2 hidden-sm hidden-xs">
-          <VolumeIcon kind={kind} />
-          <span className="co-break-word">{loc && ` (${loc})`}</span>
-        </div>
-        <div className="col-lg-2 col-md-2 hidden-sm hidden-xs">{permission}</div>
-        <div className="col-lg-2 hidden-md hidden-sm hidden-xs">
-          { _.get(pod, 'kind') === 'Pod'
-            ? <ContainerLink name={m.container} pod={pod as PodKind} />
-            : <div>{m.container}</div>
-          }
-        </div>
-      </div>
-    </React.Fragment>)}
-  </div>;
+const VolumesTableHeader = () => {
+  return [
+    {
+      title: 'Name',
+      props: { className: volumeRoColumnClasses[0]},
```

Comment:
We should be able to add sorting pretty easily now that we've moved to PF4 tables

Reply:
Added sorting to all but the type which is a derived field